### ISSUE TITLE
check referencial integrity show table-content

### ISSUE THREAD

**Issue by matthiasradde** [phpmyadmin/phpmyadmin#14598]
### Describe the bug

checking referencial integrity is not working as expected.

### To Reproduce

1. Create tables that are linked somehow (in this case use database kb4 with tables kb_docs, kb_tags and a table kb_doc_tag to link the other tables together) - you may use the same database structure I've used:
```sql
CREATE DATABASE `kb4`;
USE `kb4`;
CREATE TABLE `kb_docs` (
  `id` int(10) UNSIGNED NOT NULL,
  `modifydatetime` datetime NOT NULL,
  `subject1` varchar(255) NOT NULL,
  `subject2` varchar(255) NOT NULL,
  `details` longtext NOT NULL
) ENGINE=MyISAM DEFAULT CHARSET=latin1;
CREATE TABLE `kb_doc_tag` (
  `doc_id` int(10) UNSIGNED NOT NULL,
  `tag_id` int(10) UNSIGNED NOT NULL
) ENGINE=MyISAM DEFAULT CHARSET=latin1;
CREATE TABLE `kb_tags` (
  `id` int(10) UNSIGNED NOT NULL,
  `tag` varchar(255) NOT NULL
) ENGINE=MyISAM DEFAULT CHARSET=latin1;
ALTER TABLE `kb_docs`
  ADD PRIMARY KEY (`id`),
  ADD KEY `modifydatetime` (`modifydatetime`);
--   ADD KEY `public` (`public`);
ALTER TABLE `kb_doc_tag`
  ADD PRIMARY KEY (`doc_id`,`tag_id`);
ALTER TABLE `kb_tags`
  ADD PRIMARY KEY (`id`),
  ADD KEY `tag` (`tag`);
ALTER TABLE `kb_docs`
  MODIFY `id` int(10) UNSIGNED NOT NULL AUTO_INCREMENT;
ALTER TABLE `kb_tags`
  MODIFY `id` int(10) UNSIGNED NOT NULL AUTO_INCREMENT;

INSERT INTO `phpmyadmin`.`pma__relation` (`master_db`, `master_table`, `master_field`, `foreign_db`, `foreign_table`, `foreign_field`) VALUES
('kb4', 'docs_without_tag', 'doc_id', 'kb4', 'kb_docs', 'id'),
('kb4', 'kb_doc_tag', 'doc_id', 'kb4', 'kb_docs', 'id'),
('kb4', 'kb_doc_tag', 'tag_id', 'kb4', 'kb_tags', 'id');
```
2. Within database kb4 click on table doc_tag, click on structure, click on relation view and set relations to the other two tables.
3. Click on table doc_tag, click on Operations, scroll down and click on one of the links to check referential integrity
4. The link itself contains the correct SQL-statement as parameter sql_query, but after clicking on it I end up showing "``SELECT * FROM `kb_doc_tag` ORDER BY `doc_id` ASC``" as SQL-statement and of course showing all entries from table doc_tag

### Expected behavior

expected behaviour is to use the statement provided in parameter sql_query

### Server configuration

- Operating system: Windows 7
- Web server: Apache 2.4.34
- Database version: MySQL 5.7.23
- PHP version: 7.2.9
- phpMyAdmin version: 4.8.3

### Client configuration

- Browser: Firefox 61.0.2
- Operating system: Windows 7

**Comment by williamdes** [comment]
Successfully reproduced on demo servers.
Thanks @matthiasradde for the detailed instructions and test data !

Demo servers (root, no password)
- [`5.0.0-dev`](https://demo.phpmyadmin.net/master-config/)
- [`4.8.x`](https://demo.phpmyadmin.net/QA_4_8/)

**Comment by aschuch247** [comment]
I can reproduce the issue in phpMyAdmin 4.9.1.

Is there a chance that some magic as done in #15325 could get it working? :) `sql_query=` is present as GET parameter.

**Comment by williamdes** [comment]
@aschuch247 Yes, absolutely
I will use that same system

**Comment by williamdes** [comment]
@matthiasradde Reproduced for the 2nd time, but I edited your script.
Will be fixed in next phpMyAdmin version !

**Comment by williamdes** [comment]
@matthiasradde Fixed by e87c6b822d9a0c1750c3e1897d299b2259b0498e
You can try the patch is you want :)

**Comment by matthiasradde** [comment]
> You can try the patch is you want :)
Tested - works great :-)
Thanks to all the people involved in fixing this issue
